README.md

```diff
@@ -1,3 +1,9 @@
+# Notes on Rasa `1.x / 2.0`
+
```

Comment:
Suggest slight rewording of the first list item for clarity:
1. The master branch of this repo is compatible with Rasa Open Source **version 1.x**

Otherwise, looks good to me!